ansible/playbooks/AdoptOpenJDK_Unix_Playbook/roles/Swap_File/tasks/main.yml

```diff
@@ -50,36 +50,48 @@
 - name: Create swap file - via fallocate
   command: "fallocate -l 2G {{ swap_path }}"
   when:
-    - (swap_root.rc != 0 and swap_tmp.rc != 0) 
+    - (swap_root.rc != 0 and swap_tmp.rc != 0)
+    - ansible_distribution != "FreeBSD"
```

Comment:
it would be nice if the 'when' are the same
in this line ```- ansible_distribution != "FreeBSD"``` is on its own
but line 62 its combined